### PR TITLE
Update versions and add in override from TextInputEditText to fix extract mode hint

### PR DESCRIPTION
**Pull request by TheKeeperOfPie**
I would've also updated to AppCompatAutocompleteTextView, but that ended up breaking the background. It wasn't a necessary change to fix the extract mode, so I just kept the default one.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -13,8 +13,8 @@ POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=seatgeek
 POM_DEVELOPER_NAME=SeatGeek, Inc
 
-ANDROID_PLUGIN_VERSION=1.3.0
-BUILD_TOOLS_VERSION=21.1.2
-SUPPORT_LIBRARY_VERSION=22.2.1
-TARGET_SDK_VERSION=21
+ANDROID_PLUGIN_VERSION=2.3.3
+BUILD_TOOLS_VERSION=26.0.0
+SUPPORT_LIBRARY_VERSION=25.4.0
```
Comment (sddamico) [inline gradle.properties]: 26.0.0 support lib is out, want to just get that update over with now?

File: placesautocomplete/build.gradle [hunk 2]
```diff
@@ -40,6 +43,8 @@ android {
 
 dependencies {
     compile "com.android.support:support-v4:${SUPPORT_LIBRARY_VERSION}"
```
Comment (sddamico) [inline placesautocomplete/build.gradle]: Is this still needed?

File: placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java [hunk 3]
```diff
@@ -439,4 +443,23 @@ public void setLanguageCode(@Nullable String languageCode) {
         this.languageCode = languageCode;
         api.setLanguageCode(this.languageCode);
     }
+
```
Comment (sddamico) [inline placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java]: Won't let me comment on a line not in the review (thanks github) but can we also make this class extend from `AppCompatAutocompleteTextView`?

File: placesautocomplete/build.gradle (hunk 2 above)
Comment (TheKeeperOfPie) [inline placesautocomplete/build.gradle]: Nope, removed.

File: gradle.properties (hunk 1 above)
Comment (TheKeeperOfPie) [inline gradle.properties]: Unfortunately there are problems with the 26.0.0 release, mentioned in ASG's #supportlibwtf. An unintended dependency got pulled in, along with some style mishaps.

File: placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java (hunk 3 above)
Comment (TheKeeperOfPie) [inline placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java]: I mentioned it in the OP, but the background broke when I updated it. We don't seem to set the background anywhere, so the fix seemed non-trivial. I could spend the time debugging if we want though.

Comment (sddamico) [inline placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java]: >but the background broke

Can you describe in more detail what you mean by "broke"?

File: gradle.properties (hunk 1 above)
Comment (sddamico) [inline gradle.properties]: I read about the issues that you're describing and they shouldn't impact this project. I think we're safe to upgrade.

Comment (TheKeeperOfPie) [inline gradle.properties]: Done.

File: placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java (hunk 3 above)
Comment (TheKeeperOfPie) [inline placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java]: The background was blank and didn't have the normal EditText underline, but it seems to be a problem with the demo app not using an AppCompat theme. Fixed that.

**Pull request merged**